src/main.ts

```diff
@@ -93,7 +93,15 @@ export function injectEntry(
     return injectToArray(originalEntry, newEntry, options.entryOrder);
   }
 
-  if (originalEntry === Object(originalEntry)) {
+  if (typeof originalEntry === 'function') {

```

Comment:
Can you add tests for this?

Reply:
> Can you add tests for this?

In the latest commit I added a test for the function entry.